### ISSUE TITLE
S3Bucket validation transient issue

### ISSUE THREAD

**Issue by ktibi** [cn-terraform/terraform-aws-ecs-alb#28]
Hello @jnonino ,

Since your udpate, I have issue during sonarqube deployment : 

```
 Error: failure configuring LB attributes: InvalidConfigurationRequest: S3Bucket validation transient issue
      status code: 400, request id: 391d3af7-99cd-4cd8-9617-15557228d1ce

  with module.sonar.module.ecs_fargate.module.ecs-fargate-service.module.ecs-alb.aws_lb.lb,
  on .terraform/modules/sonar.ecs_fargate.ecs-fargate-service.ecs-alb/main.tf line 26, in resource "aws_lb" "lb":
  26: resource "aws_lb" "lb" {
```

**Comment by ktibi** [comment]
Works if I disable S3 export log from LB.

**Comment by jnonino** [comment]
Yes, seems to be an issue there. I found this while searching about it [AWS S3 Bucket for ALB Transient Issue
](https://www.reddit.com/r/Terraform/comments/t7yzh2/aws_s3_bucket_for_alb_transient_issue/). When I have some time I will try to find a way to solve it.

**Comment by jnonino** [comment]
Hi, how are you? Can you check if change in latest version fixed the issue?

**Comment by ricardf** [comment]
I'm getting exactly the same error with latest version.

Also the bucket s3 arn should be provided as output so lifecycle can be configured.

**Comment by tech-restack** [comment]
Also getting the error when using an S3 bucket in another region

│ Error: failure configuring LB attributes: InvalidConfigurationRequest: S3Bucket validation transient issue
│       status code: 400, request id: 8db33ae0-168a-42c2-bd37-109b1d2c6664

**Comment by aiplescaamdaris** [comment]
I also got that error when the bucket where the elb logs are sent had the encryption option "Server-side encryption with AWS Key Management Service keys (SSE-KMS)" with an aws managed key, and the error disappeared after switching the bucket to "Server-side encryption with Amazon S3 managed keys (SSE-S3)". 

So I think the bug/error is on AWS side with the bucket encryption and not on terraform side.
